mautrix_telegram/user.py

```diff
@@ -507,6 +504,7 @@ async def sync_dialogs(self) -> None:
         index = 0
         self.log.debug(f"Syncing dialogs (update_limit={update_limit}, "
                        f"create_limit={create_limit})")
+        await self.push_bridge_state(BridgeStateEvent.BACKFILLING)
```

Comment:
The connected state is probably going to override this after 3 seconds

Reply:
I don't see that as actually being a problem. Is there a compelling reason not to report this state for a few seconds?

Reply:
I mean it would be better to not report connected until the initial sync is actually finished, but I guess that can be fixed later (if/when we actually start using the backfilling state)

Reply:
So, the issue here is that the `CONNECTED` state is being reported too eagerly?

Reply:
Yeah, it should probably have a flag that it's backfilling and not report `CONNECTED` before it's finished (or report `BACKFILLING` in the same place where it reports `CONNECTED` when that flag is set)